components/docs-chef-io/layouts/shortcodes/automate/ha-warn.md

```diff
@@ -0,0 +1 @@
+We are currently working on making the setup and upgrade process to Automate HA a seamless experience. If you are already using Automate HA or planning to use it, please get in touch with Customer Success or your Account Manager for more information
```

Comment:
```suggestion
We are currently working on making the setup and upgrade process to Automate HA a seamless experience. If you are already using Chef Automate HA, or are planning to use it, please contact [Chef Support](https://www.chef.io/support) or your account manager for more information.
```

Reply:
This message has been suggested by @vivekvardwaz  and @nischalsrinivas . Please could you get in touch with them to get it approved.

Reply:
Hi @IanMadd ,
@nischalsrinivas has requested the below change:
```suggestion
We are currently working on making the setup and upgrade process to Automate HA a seamless experience. If you are already using Chef Automate HA, or are planning to use it, please contact your customer success manager or your account manager for more information.
```

Reply:
@IanMadd , i have done the suggested changes, please approve